src/index.js

```diff
@@ -68,15 +69,16 @@ function evalModule(str) {
 }
 
 function load({input, output}) {
-  const jssStr = localStorage.jss || example
+  const jssStr = localStorage[pkginfo.version] || example
```

Comment:
I would add something like `jss-repl-${pkginfo.version}` so that the key in local storage is not tooo generic